starter-code/public/javascripts/financial-data.js

```diff
@@ -0,0 +1,54 @@
+var url = 'http://api.coindesk.com/v1/bpi/historical/close.json'
+
+
+/* GET home page */
+  axios.get(url)
+    .then(result => {
+      //console.log(result.data.bpi)
+      printTheChart(result)
+    })
+    .catch(err => console.log(err))
+
+
+function printTheChart(something){
```

Comment:
Try to be a bit more descriptive with naming. 

'something' is not very descriptive.

A name like 'coindeskResult' would be better in this case